README.md

```diff
@@ -48,17 +48,17 @@ wizAssets.deleteFile("file://documents/settings/img/cards/card001.jpg", successC
 
 ### deleteFiles()
 
-**wizAssets.deleteFiles(Array manyURIs, Function success, Function fail );**
+**wizAssets.deleteFiles(Array localURIs, Function success, Function fail );**
 
-- delete all URIs in Array like; [ "file://documents/settings/img/cards/card001.jpg", "file://documents/settings/img/cards/card002.jpg " .. ] <br />
-- if you do not specify a filename only dir, then all contents of dir will be deleted; file://documents/settings/img/cards <br />
+- delete all URIs in Array like; [ "img/cards/card001.jpg", "img/cards/card002.jpg " .. ] <br />
+- if you do not specify a filename only dir, then all contents of dir will be deleted; img/cards <br />
 - the array CAN contain one URI string
 
 ### getFileURI()
 
-**wizAssets.getFileURI(String filePathWithFilename, Function success, Function fail);**
+**wizAssets.getFileURI(String localURI, Function success, Function fail);**
 
-- A success returns URI string like file://documents/settings/img/cards/card001.jpg <br />
+- A success returns a file URI string like file://documents/settings/img/cards/card001.jpg <br />
```

Comment:
Hmm, it looks like it's not quite clear. User gives a local URI and gets back a file URI ```file://...``` which can be used for example to load an image using ```image.src = fileURI```.
```wizAssets://...``` is not implemented yet so user still needs to get back the file URI pointing to his assets to manipulate it.